debezium-connector-oracle/src/main/java/io/debezium/connector/oracle/OracleOffsetContext.java

```diff
@@ -277,4 +336,29 @@ else if (scn != null) {
         }
         return null;
     }
+
+    /**
+     * Helper method to read the in-progress transaction map from the offset map.
+     *
+     * @param offset the offset map
+     * @return the in-progress transaction map
+     */
+    public static Map<String, Scn> loadSnapshotPendingTransactions(Map<String, ?> offset) {
+        Map<String, Scn> snapshotPendingTransactions = new HashMap<>();
```

Comment:
Could we possibly store this similar to how we do the collections to be snapshotted via incremental snapshots?  There rather than store each collection separately with some key prefix, instead, there is a single entry and the values are stored as a string separated by a `,`.  I believe we could effectively do the same here, no?